cert/CACreationScript.sh

```diff
@@ -18,6 +18,10 @@
 
 echo Run this script from the directory containing the CBRS openssl.cnf file present in it. The openssl command must be present on the path.
 
+echo Usage: ./CACreationScript.sh
+echo Usage: ./CACreationScript.sh pkey cbsd_ee.pkey
```

Comment:
I'm unclear what these variants do. It looks like it's saving a particular key file to a target before running the rest of the script? Couldn't this just be done in the command line before launching the creation script?

Reply:
These variants are added to support End Entity Certificate generation for CBSD device using pre-defined RSA key or CSR. If RSA key is preconfigured in CBSD device and device can not use any new key,hence this variant shall support those cases.
While running the script user can provide RSA key file or CSR for CBSD End Entity certificate generation.

Reply:
Ah, I see. Maybe add a note about that use case.